claat/main.go

```diff
@@ -101,6 +107,16 @@ func main() {
 	os.Exit(exitCode)
 }
 
+// parsePassMetadata parses metadata fields to parse that are not explicitly handled elsewhere.
+// It expects the fields to be passed in as a comma separated list (extraneous spaces are autoremoved), and returns a set of strings.
+func parsePassMetadata(passMeta string) map[string]bool {
+	fields := make(map[string]bool)
```

Comment:
nit: since opted for non-`make` inits, change to literal.
GTG

Reply:
Done